flit/install.py

```diff
@@ -22,6 +25,32 @@
     'prefix'  : sys.prefix,
 }
 
+
+# Requires-Dist parsing from pkg_resources
+
+_EQEQ = re.compile(r"([\(,])\s*(\d.*?)\s*([,\)])")
+def _requires_dist_to_requirement(requires_dist):
+    """Parse 'Foo (v); extra == b' from Requires-Dist
+
+    return pip-style 'foo[x]==v'
+
+    Evaluates environment markers if any is present.
+    Returns None if environment markers exclude the dependency.
+    """
+    parts = requires_dist.split(';', 1) + ['']
+    distvers = parts[0].strip()
+    mark = parts[1].strip()
+    distvers = re.sub(_EQEQ, r"\1==\2\3", distvers)
+    distvers = distvers.replace('(', '').replace(')', '')
+    if not mark:
+        return distvers
+    # if an environment marker is present, check it:
+    from _markerlib import compile as compile_marker
```

Comment:
This is part of setuptools, so I don't want to import it. It should be OK to leave environment markers alone for requirements.txt and let pip handle them.

I think I'm going to write my own version of this function. It's taking me far more effort than it should to verify that it correctly does a relatively simple thing.